### ISSUE TITLE
Installation on VS Code not working?

### ISSUE THREAD

**Issue by trylks** [scalameta/metals#1772]
**Question**

I am using VS Code on Linux. I have installed Metals with the one click installation, several times, some of them recommended by the prompt in VS Code.

After restarting the laptop, Metals does not seem to be working, though. For example

* I can run scalafmt from the terminal, but not from VS Code (ctrl + shift + I), I get an error saying that no formatter is installed for Scala
* In the command palette I can find `Metals: Run doctor`, but again I get an error message (as in the screenshot)

This may or may not be a bug. I guess it is hard to screw up in the one click installation, but I feel capable of that.

**Screenshots**

![Screenshot_2020-05-21_22-03-55](https://user-images.githubusercontent.com/1286855/82606825-0649a980-9bb8-11ea-8cdb-93315de8a413.png)

**Installation:**

- Operating system: Linux
- Editor: Visual Studio Code
- Metals version: v0.9.0

**Additional context**

I have uninstalled all Scala related plugins, restarted the laptop, reinstalled Metals,... nothing seems to help.

**Search terms**

Installation problems

**Comment by tgodzik** [comment]
Thanks for reporting! This looks like the extension is failing to be activated. Do you get anything in the Metals output? 

![image](https://user-images.githubusercontent.com/3807253/82652515-07152680-9c1e-11ea-9473-87967642a38a.png)

Or maybe in the extension log?

![image](https://user-images.githubusercontent.com/3807253/82652558-14caac00-9c1e-11ea-9f21-60a7775a9dd7.png)

Usually this connected to either a problem with java version or that the server failed to download.

**Comment by trylks** [comment]
tl; dr:
1. Problem solved, thank you very much, please feel free to close this
1. Documentation could say: "do not have a `.dotty-ide-artifact` in your project"
1. An example project `println("hello world")` with sbt and everything (for the doctor) could help noobs

I am returning to Scala-land after some years of Pythonista, and also starting with VS Code. So I decided to try the latest (why not?): https://github.com/lampepfl/dotty-example-project

Somehow that created a `.dotty-ide-artifact` file, which prevented metals from starting (and installing), as shown in the output, after removing it, everything works. Therefore, even if Metals and Dotty are compatible, they are not going to be compatible in the same project (which maybe should be noted in the documentation, where it says that the plugins are compatible).

Still the `build.sbt` in the abovementioned example project is not as good as Metals would like (I appreciate the help, very needed with my Scala significantly rusty), so I will be working on that. Perhaps an example project would be useful, to start on the right track. Features of a rich IDE like Metals, and example projects, are both useful for noobs (like I consider myself now).

**Comment by tgodzik** [comment]
Glad it worked!

> 3. An example project `println("hello world")` with sbt and everything (for the doctor) could help noobs

We are actually working on that here: https://github.com/scalameta/metals/pull/1728
It will allow users to create one of many example projects.

> Somehow that created a .dotty-ide-artifact file, which prevented metals from starting (and installing), as shown in the output, after removing it, everything works. Therefore, even if Metals and Dotty are compatible, they are not going to be compatible in the same project (which maybe should be noted in the documentation, where it says that the plugins are compatible).

We are actually moving away from the separate Dotty plugin and at some point it will be deprecated, as Metals already provides support for Dotty. `.dotty-ide-artifact` is probably generated when starting via sbt, but not sure. But going forward, we will be recommending only Metals for VS Code.
